hcctl/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "hcctl"
+version = "0.1.0"
+authors = ["Iain Earl <iain@itmecho.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+healthchecks = { path = "../healthchecks", version = "^1.0.2-alpha.0"}
+anyhow = "1.0.32"
+structopt = "0.3.18"
```

Comment:
We use `clap` in the `monitor` binary and `structopt` is a wrapper around it so I'd rather use it directly. `monitor` also includes a simple `std::env::var` backed config for loading from env that I also prefer over `structopt`.